appinventor/components/src/com/google/appinventor/components/runtime/Form.java

```diff
@@ -783,6 +784,38 @@ public String AboutScreen() {
   public void AboutScreen(String aboutScreen) {
     this.aboutScreen = aboutScreen;
   }
+  
+  /**
+   * ShowStatusBar property getter method.
+   *
+   * @return  showStatusBar boolean
+   */
+  @SimpleProperty(category = PropertyCategory.APPEARANCE,
+      description = "The status bar is the toppest bar on the screen. You have the option to hide the status bar. This function returns you the visibilty of the status bar.")
```

Comment:
Change description to:
The status bar is the topmost bar on the screen.  This property reports whether the status bar is visible.